Update README.md (#70)
IMAP-Science-Operations-Center/imap-data-access: README.md

This lightweight  Python package allows users to download, query, and upload data from the IMAP Science Data Center (SDC).

> For more information and detailed instructions see https://imap-processing.readthedocs.io/en/latest/data-access-api/index.html

Reviewer: I don't think markdown should use bare urls.
```suggestion
For more information see the [detailed instructions](https://imap-processing.readthedocs.io/en/latest/data-access-api/index.html).
```

imap-data-access -h
```

### Base Command Arguments
```bash
$ imap-data-access query # or
$ imap-data-access download # or
$ imap-data-access upload
```

> Run '-h' with any base command for more information

Reviewer: ```suggestion
Add the '-h' help flag with any base command for more information
```

> Add the '-h' help flag with any base command for more information

Reviewer 2: Great idea to include the bear-bones usage here for quick reference. I think between this and the link to the more detailed instructions is a good balance.

Download a level 0 SWE file on 2024/01/05

> Note: Make sure the directory configuration is correct, see Data Directory section for more information.

Reviewer: You can add a warning/tip admonition here. https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Also, if you are indicating a user should look at a section, it is helpful to link out to that section directly.
```suggestion
> [!NOTE]
> Make sure the directory configuration is correct, see the [Data Directory section](https://github.com/IMAP-Science-Operations-Center/imap-data-access#data-directory) for more information.
```
